src/main/java/omg/group/priuttelegrambot/dto/chats/ChatsDto.java

```diff
@@ -0,0 +1,14 @@
+package omg.group.priuttelegrambot.dto.chats;
+
+import lombok.Data;
+
+import java.time.LocalDateTime;
+
+@Data
+public abstract class ChatsDto {
+    private Long id;
+    private Boolean isChatting;
```

Comment:
```suggestion
    private Boolean chatting;
```
на is называем методы, которые возвращают boolean